api/tests/test-helper.js

```diff
@@ -133,6 +143,7 @@ function generateAuthenticatedUserRequestHeaders({
     authorization: `Bearer ${accessToken}`,
     'x-forwarded-proto': protoHeader,
     'x-forwarded-host': hostHeader,
+    cookie: `locale=${locale}`,
```

Comment:
Faire comme le acceptLanguage pour éviter le `locale=undefined` :

```suggestion
    ...(cookie && { 'cookie': `locale=${locale}` }),
```